src/front-end/typescript/lib/pages/organization/edit/tab/team.tsx

```diff
@@ -160,6 +165,32 @@ const update: Update<State, Msg> = ({ state, msg }) => {
       ];
     }
 
+    case 'approveAffiliation':
+      return [
+        state
+          .set('approveAffiliationLoading', msg.value.id)
+          .set('showModal', null),
```

Comment:
For Brianna's learning: Why is this inside the return here vs. how it's done on line 195?

Reply:
This code was brought over from the organizations page 
`src/front-end/typescript/lib/pages/user/profile/tab/organizations.tsx` and modified.  I hadn't noticed the difference in pattern. I don't believe it causes a change in behaviour. Would you preffer format them the same?